mediator/med.F90

```diff
@@ -916,6 +917,20 @@ subroutine AdvertiseFields(gcomp, importState, exportState, clock, rc)
        end if
     end do ! end of ncomps loop
 
+    ! Should mediator check for NaNs?
```

Comment:
should there be a isPresent, isSet check here?

Reply:
Sure I can add that and default to false if not present and set.  Thanks